tools/platform.sh

```diff
@@ -24,7 +24,7 @@ case "$OSTYPE" in
     linux-*)
         if have-command cmd.exe; then
             _is_wsl=true
-            _is_windows=true
+            _is_unix_like=true
```

Comment:
Update comment at the top of the file `(i.e. $IS_WSL and $IS_WINDOWS)` to `(i.e. $IS_WSL and $IS_UNIX_LIKE`).